models/adlink.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+)
+
+type AdLink struct {
+	Base
+
+	Result		    *Result  `orm:"rel(fk)"`
+	Type            string   `orm:"type(text)"`
+	Position        string   `orm:"type(text)"`
+	Link            string   `orm:"type(text)"`
+}
+
+func init() {
+	orm.RegisterModel(new(AdLink))
+}
+
+// TableName set the custom table name to plural because the default table name is singular
+func (a *AdLink) TableName() string {
+	return "ad_links"
+}
+
+// CreateAdLink insert a new AdLink into database and returns last inserted Id on success.
+func CreateAdLink(adLink *AdLink) (id int64, err error) {
```

Comment:
Should we update the return parameter to be unnamed? This will align with our conventions. If yes please also check other functions too.
![Screen Shot 2564-05-14 at 12 17 31](https://user-images.githubusercontent.com/29707647/118224970-5d721980-b4ae-11eb-8028-29132901d183.png)

Reply:
@gutakk fixed in [669f16a](https://github.com/carryall/go-google-scraper-challenge/pull/42/commits/669f16a5d95b1f3e5cd9ef5ac99dec00ec3d6277)